source/open-definition-revision-process-dev.markdown

```diff
@@ -2,19 +2,21 @@
 Definition Revision Approval Process _- draft_
 ====================================
 
-1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.
+1. Issues with the current Open Definition are identified.  The Open Definition community will disucss on the mailing list or discuss.okfn.org forum and decide which issues are to be dealt with in the next release.
 
 2. A new draft document is started.  Chair notifies the list and other relevant fora that a new draft document has been started summarizing the issues that the upcoming release is to address and inviting participation.
 
-3. Issues with the current Open Definition are identified.  The Open Definition community will discuss on the mailing list and decide which issues are to be dealt with in the next release reach consensus on each of these issues.  The draft document is updated to relfect current consensus.
```

Comment:
Note: this was a totally redundant item to item 1, like it was just a typo when someone moved it and forgot to delete the old lines or something.